File: NoteClient/src/app/models/note.ts
import { INoteHttp } from '../models/models-http/note.http';
import { from } from "rxjs";
import { INote } from './note.interface';

export class Note implements INote {
    id: number;
    title: string;
    text: string;
    dateInit: Date;
    dateUpdate: Date;
    isActual: boolean;
}

File: NoteManager.API/NoteManager.Core/Models/Note.cs
using System;

namespace NoteManager.Core.Models
{
    public class Note
    {
        public long Id { get; set; }
        public string Title { get; set; }
        public string Text { get; set; }
        public DateTime DateInit { get; set; }
        public DateTime DateUpdate { get; set; }
        public bool IsActual { get; set; } = true;
    }
}